k-distribution/tests/regression-new/ceil-map-generated/ceil-map-cellMap-spec.k.out

```diff
@@ -0,0 +1,50 @@
+  #Not ( {
+    partialInt ( _K1 )
+  #Equals
+    partialInt ( _K2 )
+  } )
+#And
```

Comment:
Yes, this is missing all of the definedness conditions for the keys and elements of the map. The rest of the map, `_CELLMAP`, doesn't need such a condition because it's an element variable.